docs/kyma/docs/04-04-gke-installation.md

```diff
@@ -34,15 +33,15 @@ By default, Kyma is installed on a GKE cluster with a wildcard DNS provided by [
 3. Install Tiller on your GKE cluster. Run:
 
     ```
-    kubectl apply -f installation/resources/tiller.yaml
+    kubectl apply -f https://raw.githubusercontent.com/kyma-project/kyma/{RELEASE_TAG}/installation/resources/tiller.yaml
     ```
 
 4. Add your account as the cluster administrator:
     ```
     kubectl create clusterrolebinding cluster-admin-binding --clusterrole=cluster-admin --user=$(gcloud config get-value account)
     ```
 
-## DNS setup and TLS certificate generation
+## (Optional) DNS setup and TLS certificate generation
```

Comment:
I don't like the idea of putting a word in parenthesis in the heading. We have notes that state that some steps are optional. Additionally, I think we're going to restructure these docs using panels.

Reply:
notes are long, and indication in the heading that the particular step is optional is very helpful if you scan the content of the document quickly. Especially that now with DNS option or xip the structure got a bit more complex